### ISSUE TITLE
The deployers do not work properly

### ISSUE THREAD

**Issue by Tiburan1** [Laskyyy/Create-Astral#118]
I'm trying to create a tunneling machine with the drills and the deployers and some of them work a couple of uses and they stop working and some of them don't work directly. # #

**Comment by Nilixen** [comment]
Are you sure that they're not in claimed chunks? Cuz' of 2.0.4c I've noticed that deployers don't work in claimed chunkes unless you modify ftbchunks config

**Comment by Tiburan1** [comment]
Yes, I removed all the claimeds chunks, as I saw that could be the reason

**Comment by jonasogren** [comment]
I can also say that Deployers, in a moving contraption created either by a Mechanical bearing or a Minecart assembler, does not work since moving from version 2.0.3 to 2.0.4 - In version 2.0.3 it works fine.

**Comment by Erdragh** [comment]
Was this resolved?

**Comment by Sensei-Noah** [comment]
I can confirm that deployers in a contraption don't work while being in a claimed chunk. And unclaiming the chunk fixes the issue.

**Comment by Erdragh** [comment]
or allowing fake players in your claims:

```
/ftbteams server settings [teamname] ftbchunks:allow_fake_players allow"
```

or

press `]` (opens ftbchunks map) --> large map (top left) --> bottom right (server settings)
